monkey/common/events/stolen_credentials_event.py

```diff
@@ -0,0 +1,18 @@
+from dataclasses import dataclass
+from typing import Sequence
+
+from common.credentials import Credentials
+
+from . import AbstractEvent
+
+
+@dataclass(frozen=True)
+class StolenCredentialsEvent(AbstractEvent):
+    """
+    An event that was initiated/observed when an agent collects credentials from the victim.
```

Comment:
```suggestion
    An event that is initiated/observed when an agent collects credentials from the victim
```

Reply:
Maybe something like
```suggestion
    An event that occurs when an agent collects credentials from the victim
```
or
```suggestion
    An event that is triggered when an agent collects credentials from the victim
```